Source Main 5.2/source/ZzzScene.cpp

```diff
@@ -2498,6 +2499,11 @@ void MainScene(HDC hDC)
                 {
                     break;
                 }
+                // reduce CPU usage by yielding during spin phase
+                if (spin_ms - spinned_ms > spin_yield_threshold_ms)
+                {
+                    std::this_thread::yield();
+                }
```

Comment:
I think this is necessary for cases where the sleep threshold was not reached, to prevent busy-spinning for almost 8ms